src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -4,38 +4,18 @@
 LGPL2.1.  See file COPYING.
 """
 
-import errno
 import logging
 import os
+import errno
 
 import cephfs
-import rados
 
+from subvolspec import SubvolumeSpec
+from exception import VolumeException
```

Comment:
s/exception/.exception/

There is a python inbuilt module called 'exceptions'.  I think it'll be better if you rename the 'exception' module as 'volume_exception'.

Reply:
it's pretty normal to name it as `exception.py` imo.

Reply:
in the sense, it's common to stash various custom exception classes into one exception source.